plugins/reach/lib/model/data/VendorLiveCaptionCatalogItem.php

```diff
@@ -12,19 +12,27 @@ public function applyDefaultValues()
 
 	public function getTaskJobData($object)
 	{
+		$latestEvent = null;
 		if($object instanceof LiveEntry && $object->hasCapability(LiveEntry::LIVE_SCHEDULE_CAPABILITY))
 		{
 			$events = ScheduleEventPeer::retrieveByTemplateEntryIdAndTypes($object->getId(), [ScheduleEventType::LIVE_STREAM]);
 
-			$data = new kScheduledVendorTaskData();
```

Comment:
before, we returned empty kScheduledVendorTaskData when there were no events on the entry
now, we only return filled kScheduledVendorTaskData when we find events and null otherwise

was this intended?

Reply:
@salma2222  fixed.